Changelog.md

```diff
@@ -20,6 +20,7 @@ Bugfixes:
  * Source Locations: Properly set source location of scoped blocks.
  * Standard JSON: Properly allow the ``inliner`` setting under ``settings.optimizer.details``.
  * Type Checker: Fix internal compiler error related to having mapping types in constructor parameter for abstract contracts.
+ * Code Generator: Fix internal error when super would have to skip an unimplemented function in the virtual resolution order.
```

Comment:
The list is currently still sorted, I'd try to keep it sorted

Reply:
Sorry. Fixed.